README.md

```diff
@@ -14,4 +14,26 @@ If you have forked this repository and would like to use this as a codespace tem
 4. Click "Use this template" button at the top right
 5. Click "Open in a codespace"
 
-If you need to delete the codespace, you can do this at [https://github.com/codespaces](https://github.com/codespaces)
+## FAQ - Frequently Asked Questions
+
+Can I see a demo video?
+- Link to the demo video for the Actions Playground: https://github.com/github/octogrowth/discussions/921#discussioncomment-4936834)
```

Comment:
Question: Is this readme for internal users? This points to our team repo. This is fine for internal users, but if the target audience is external (non-staff) users, this won't work.